data/en/mbs_0201.json

```diff
@@ -371,7 +379,7 @@
         },
         {
             "name": "trad_as",
-            "validator": "optional_string"
+            "validator": "string"
```

Comment:
How come mbs_0201.json needs trad_as to have not have the optional_string?

Reply:
Fixed! Good spot.